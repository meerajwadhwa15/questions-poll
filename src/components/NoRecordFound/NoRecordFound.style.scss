.no-record-found {
    color: $error-color;
    text-align: center;
    margin: 0 auto;
}