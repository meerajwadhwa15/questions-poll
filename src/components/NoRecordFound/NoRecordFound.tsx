import React, { FunctionComponent, ReactNode } from "react";

export type Props = {
  children: ReactNode;
};

const NoRecordFound: FunctionComponent<Props> = ({ children }) => (
  <div className="no-record-found">{children}</div>
);

export default NoRecordFound;
