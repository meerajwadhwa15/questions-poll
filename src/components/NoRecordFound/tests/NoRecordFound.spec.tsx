import React from "react";
import NoRecordFound from "../NoRecordFound";
import { create } from "react-test-renderer";

const props = {
};

test("Matches the snapshot with NoRecordFound", () => {
  const component = create(
      <NoRecordFound {...props}>No Record Found.</NoRecordFound>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
