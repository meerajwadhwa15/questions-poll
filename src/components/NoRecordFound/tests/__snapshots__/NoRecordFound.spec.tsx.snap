// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot with NoRecordFound 1`] = `
<div
  className="no-record-found"
>
  No Record Found.
</div>
`;
